common/init.rc

```diff
@@ -137,6 +137,18 @@ on property:sys.interactive="active"
     chmod 0660 /sys/devices/system/cpu/cpufreq/interactive/boostpulse_duration
     chown system system /sys/devices/system/cpu/cpufreq/interactive/io_is_busy
     chmod 0660 /sys/devices/system/cpu/cpufreq/interactive/io_is_busy
+#For usb
+    write /sys/class/gpio/export 117
+    write /sys/class/gpio/gpio117/direction "out"
+    write /sys/class/gpio/gpio117/value 0
+    chmod 777 /sys/class/gpio/gpio117/value
+
+#FP Sensor STAT
+    write /sys/class/gpio/export 58
+    write /sys/class/gpio/gpio58/direction "in"
+    chmod 777 /sys/class/gpio58/value
+    chmod 777 /dev/video0
+    chmod 777 /dev/spidev1.0
 
```

Comment:
This is wrong, you should never change common/init.rc as it is used by other devices.

Plus your changes also are in the cid-specific init.rc, please remove that change.